### PR TITLE
Add Component ID of QX1 gimbal

### PR DESCRIPTION
**Pull request by bugobliterator**

### CODE REVIEW

**Comment by magicrub** [comment]
There are two changes in this PR.

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -377,7 +377,7 @@
                <entry value="153" name="MAV_COMP_ID_SERVO14">
                     <description/>
                </entry>
-               <entry value="154" name="MAV_COMP_ID_GIMBAL">
+               <entry value="154" name="MAV_COMP_ID_SOLO_GIMBAL">
```
Comment (lvale) [inline message_definitions/v1.0/common.xml]: I believe this ID was not created for the Solo Gimbal, but for generic Mavlink aware Gimbal controllers like the STorM32

Comment (rmackay9) [inline message_definitions/v1.0/common.xml]: Yes @lvale is right.  It was meant for use by any mavlink enabled gimbal.

Comment (bugobliterator) [inline message_definitions/v1.0/common.xml]: @rmackay9 OK, I will change it back to MAV_COMP_ID_GIMBAL

**Comment by bugobliterator** [comment]
@rmackay9 @magicrub I have reverted the change to MAV_COMP_ID_GIMBAL. This PR now just adds a new component called QX1_GIMBAL

**Comment by magicrub** [comment]
looks good to me

**Comment by bugobliterator** [comment]
@tridge @rmackay9 can you pull in this PR?

**Comment by tridge** [comment]
applied, thanks!

**Pull request merged**